server/controllers/DataAccessLayer.js

```diff
@@ -330,7 +330,7 @@ GetChatLogString = function(chatMessages) {
     // For each ChatMessage in the given Report's chatLog
     for (let i = 0; i < chatMessages.length; i++) {
         // Create a unique string for the message
-        let chatMessageString = "{" + chatMessages[i].username + "," + chatMessages[i].message + "}";
+        let chatMessageString = "{" + chatMessages[i].userID + "," + chatMessages[i].message + "}";
```

Comment:
Where is this ID being retrieved from? I've made some changes around chat ids so might be good to take a look at my PR

Reply:
The ID comes from the ChatMessage object, which gets the relevant id from the chat logs

Reply:
Nevermind, we're all good then